dissect/fat/exfat.py

```diff
@@ -86,34 +86,36 @@ def sector_to_cluster(self, sector):
 
     def runlist(self, starting_cluster, not_fragmented=True, size=None):
         """
-        Creates a RunlistStream compatible runlist from exFAT FAT structures
+        Creates a RunlistStream compatible runlist from exFAT FAT structures, in sectors.
 
         Args:
             starting_cluster (int): First cluster of file, folder or location in question
 
         Returns:
-            runlist: [(sector_offset, run_length)]"""
+            runlist: [(sector_offset, run_length_in_sectors)]"""
 
         # If file is not fragmented clusters will not be present in the FAT
         cluster_chain = [starting_cluster] if not_fragmented else self.get_cluster_chain(starting_cluster)
         runlist = []
+        sectors_per_cluster = 2**self.vbr.sectors_per_cluster_exp
 
         # TODO Graceful way to construct a runlist of non-fragmented streams spanning multiple sectors
         if size:
-            run = -(-size // self.cluster_size)
+            run = -(-size // self.cluster_size) * sectors_per_cluster
```

Comment:
The coverage checker is complaining about this branch. Not much to be done here, this code is unused.

Reply:
That’s fine for now. This specific file system implementation is due for a rewrite since it’s not compatible anymore with the rest of the Dissect API.